theano/tensor/elemwise.py

```diff
@@ -502,6 +503,7 @@ class Elemwise(OpenMPOp):
     | Elemwise(log)(rand(3, 4, 5))
 
     """
+    __props__ = ("scalar_op", "inplace_pattern", "name", "nfunc_spec", "openmp")
```

Comment:
The right fix would be to support dict in props. I tried and it wasn't easy, so I let this to later. For this, we would need frozendict and this isn't in the default python. I'm not sure it is good to add extra dependencies. So just remove it.